faq/templatetags/faq_inclusion_tags.py

```diff
@@ -0,0 +1,8 @@
+from django import template
+
+register = template.Library()
+
+
+@register.inclusion_tag("faq/inclusion_tags/faq_detail.html")
```

Comment:
Redundant. Remove the entire templatetags directory. I must still remove it from jmbo-post.